packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -78,13 +76,42 @@ const SupportListItem = ({ item }: { item: SupportItem }) => {
   );
 };
 
-export const SupportButton = ({ children }: PropsWithChildren<Props>) => {
-  const { items } = useSupportConfig();
+const createDisplayedItems = (
+  showCondition: Boolean | undefined,
+  generalItems: SupportItem[],
+  supportItems: SupportItem[] | undefined,
+) => {
+  if (showCondition === undefined) return generalItems;
+  else if (showCondition)
+    return supportItems && [...supportItems, ...generalItems];
+  return supportItems;
+};
+
+type Props = {};
 
+type SupportButtonProps = {
+  title?: string;
+  supportItems?: SupportItem[];
+  showBothLists?: Boolean | undefined;
```

Comment:
Sorry, I probably wasn't super clear before :( , but when you use optional `?` it implicitly means the type can also be undefined. So explicitly adding the type undefined to the union isn't necessary here.